README.rst

```diff
@@ -97,5 +97,4 @@ Official Resources
 * `training.plone.org <https://training.plone.org/>`_ - Training classes for developers/integrators/users/designers.
 * `plone.api <https://docs.plone.org/develop/plone.api/docs/index.html>`_ - Documentation for plone.api.
 * `plone.restapi <https://plonerestapi.readthedocs.io/en/latest/>`_ - Documentation for plone.restapi.
-* `official Discord chat <https://discord.gg/w8e5WCAKGs>`_ - Join link.
-
+* `official Discord chat <https://discord.gg/zFY3EBbjaj>`_ - Join link.
```

Comment:
Wouldn't it be better to direct people to plone.org/chat?

Reply:
I decided to update the link, but I would say we need a better README for `Plone` and `Products.CMFPlone`.
Maybe next week if @rioksane has time